src-tauri/risuko-webview-upgrade/android/src/main/java/app/risuko/webview_upgrade/sandbox/SandboxedProcessServiceDelegate.kt

```
@@ -0,0 +1,422 @@
+    }
+
+    private fun patchContextImplClassLoaderField(ctx: Context, cl: ClassLoader) {
+        var base: Context = ctx
+        while (base is ContextWrapper) base = base.baseContext
+        val implClass = Class.forName("android.app.ContextImpl")
+        if (!implClass.isInstance(base)) return
+        val f = implClass.getDeclaredField("mClassLoader").apply { isAccessible = true }
+        f.set(base, cl)
+    }
+
+    private fun injectDex(
```

detekt (1.23.8)

[warning] The function injectDex appears to be too complex based on Cyclomatic Complexity (complexity: 34). Defined complexity threshold for methods is set to '25'

(detekt.complexity.CyclomaticComplexMethod)